src/client/health.go

```diff
@@ -1,13 +1,20 @@
 package client
 
 import (
-	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
-
-	"github.com/gogo/protobuf/types"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"google.golang.org/grpc/health/grpc_health_v1"
 )
 
 // Health health checks pachd, it returns an error if pachd isn't healthy.
 func (c APIClient) Health() error {
-	_, err := c.healthClient.Health(c.Ctx(), &types.Empty{})
-	return grpcutil.ScrubGRPC(err)
+	var req grpc_health_v1.HealthCheckRequest
+
+	response, err := c.healthClient.Check(c.Ctx(), &req)
+	if err != nil {
+		return errors.Errorf("health check errored %w", err)
```

Comment:
I don't think so, as the error no longer comes from grpc

Reply:
The wrapped `err` does though, right?  `c.healthClient` is `grpc_health_v1.NewHealthClient(clientConn)`.

The intent of ScrubGRPC seems to be to remove the status code from the message, and remove the type information so that a cast to `interface { GRPCStatus() *status.Status }` can't succeed.  Not really sure why, but there is probably a reason.  A scary reason.